workflow/deploy.md

```diff
@@ -51,16 +51,12 @@ In order to deploy to Kubernetes you will need to create 2 Jenkins jobs on the w
 
 Update the **apply** job:
 
-- Go to the [configure page](https://jenkins.canonical.com/webteam/job/apply-production-deployment-configs/configure)
-- Add these lines:
+- If you are going to deploy a new site, check that the file `/sites/<DNS_OF_THE_WEBSITE>.yaml` exists in the [deployment configs repository](https://github.com/canonical-web-and-design/deployment-configs)
+- Go to the [staging configuration](https://jenkins.canonical.com/webteam/job/apply-staging-deployment-configs/configure) and [production configuration](https://jenkins.canonical.com/webteam/job/apply-production-deployment-configs/configure) on Jenkins.
+- Add this line at the end of both build scripts:
 
 ```bash
-export TAG_TO_DEPLOY=$(kubectl get deployment <DNS_OF_THE_WEBSITE_WITH-DASHES> --namespace production --output jsonpath="{.spec.template.spec.containers[*].image}" | grep -P -o '(?<=:)[^:]*$')
-envsubst < services/<DNS_OF_THE_WEBSITE>.yaml | kubectl apply --namespace production --filename -
-
-[...]
-
-kubectl apply --namespace production --filename ingresses/production/<DNS_OF_THE_WEBSITE>.yaml
+deploy <DNS_OF_THE_WEBSITE>
 ```
 
 Create the **staging** job:
```

Comment:
Could you also change these instructions to talk about `{DOMAIN_NAME}-staging` and `{DOMAIN_NAME}-production` rather than `deploy-to-<STAGING_DNS>` etc.?

Reply:
@nottrobin is it fine now?

Reply:
Yep all looks good, but per [the rules](https://github.com/canonical-web-and-design/practices/blob/master/CONTRIBUTING.md#approval-criteria) you need one more :+1:

Reply:
Oh, I didn't know about these rules for this repo. Okay I will get someone. Thank you!

Reply:
Yep, just this repo is different